src/engraving/utests/playbackeventsrendering_tests.cpp

```diff
@@ -360,9 +360,9 @@ TEST_F(PlaybackEventsRendererTests, SingleNote_Turn_Regular)
     int expectedSubNotesCount = 4;
     duration_t expectedDuration = QUARTER_NOTE_DURATION / expectedSubNotesCount;
     pitch_level_t nominalPitchLevel = pitchLevel(PitchClass::F, 4);
-    std::vector<pitch_level_t> expectedPitches = { nominalPitchLevel + PITCH_LEVEL_STEP,
+    std::vector<pitch_level_t> expectedPitches = { static_cast<short>(nominalPitchLevel + PITCH_LEVEL_STEP),
```

Comment:
'short' is a local implementation of int_fast_16 on your machine. You can't just cast it here, otherwise it would work only for you. I suggest to get a local variable with pitch_level_type. Like nominalPitchLevel one

Reply:
See #10188